static/js/side-navigation.js

```diff
@@ -0,0 +1,33 @@
+(function() {
```

Comment:
Is this JS needed? I think we should make everything work without JS first as we need to figure out how we want to structure and make the JS maintainable with tests.